terraform/appflask.py

```diff
@@ -828,13 +822,27 @@ def calico_nodes_view():
                                 ipv4_cluster_subnet=ipv4_cluster_subnet,
                                 ipv6_cluster_subnet=ipv6_cluster_subnet,
                                 calico_nodes=json.dumps(calico_nodes, indent=4),
-                                hostnames = hostnames,
+                                hostnames = get_hostnames(calico_nodes),
                                 manage_cluster=manage_cluster,
                                 fabric_type=fabric_type)
 
 
+def stream_calico_nodes_update(calico_nodes):
+    return turbo.stream([
```

Comment:
I think we should do if turbo can stream then stream

Reply:
The question is what would happen if turbo can't stream.  Does the whole thing just crash?

Reply:
Pretty much but at least we know something is wrong and we don't offer back stale/broken data. I prefer to fail hard then to silently fail. 

Yes Ideally we should handle this in a smarter way :D

Reply:
Okay I'll make the change. 👍